edge/pkg/common/message/message.go

```diff
@@ -7,13 +7,18 @@ import (
 //constant defining node connection types
 const (
 	ResourceTypeNodeConnection = "node/connection"
-	OperationNodeConnection    = "publish"
```

Comment:
Please check it to see whether we can change it to "connect".

Reply:
I tested it's okay.

Reply:
Put operation val togerther.

Reply:
I'd suggest you split this change in another pr as a bug fix or something others.  And avoid modify too much in one PR :) @muxuelan